core/templates/dev/head/css/about.css

```diff
@@ -0,0 +1,369 @@
+/*
+  Copyright 2018 The Oppia Authors. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/*
+  Animation keyframes. These must be placed at the top of the file
+  in order to work in IE.
+*/
+@-webkit-keyframes dot {
```

Comment:
Why are these specific to the About page? Are they not used in other pages?

Reply:
These lines have been used in oppia.css too though I'm not sure if they are required here.

Reply:
You can check by removing. But more than that, you should analyze what those lines mean, figure out where they are used, and then act accordingly.

Reply:
Okay, got it. Thanks!

Reply:
Regarding these lines: We need keyframes to control the intermediate steps of the CSS animation sequence for various browsers and so we need to keep them here.

Reply:
I understand why you need keyframes. Why do these need to be in this file rather than in oppia.css?

The keyframes here define an animation type, "dot". Is that animation type specific to something in the About page?

Reply:
Oh, my bad! This is more generally used throughout the site. These should not be placed here if there already present in oppia.css.  
I'll remove them.